openid-4-verifiable-presentations-1_0.md

```diff
@@ -296,16 +301,75 @@ GET /authorize?
   &nonce=n-0S6_WzA2Mj HTTP/1.1
 ```
 
-The following is a non-normative example of an Authorization Request with a `request_uri_method` parameter (including the additional `client_id_scheme` and `client_metadata` parameters): 
+The following is a non-normative example of Authorization Request with request object as value:
+```
+GET /authorize?
+  response_type=vp_token
```

Comment:
response_type isn't legal here, when `request=` is used response_type can only be inside the request object, it should be removed:
```suggestion
```

Reply:
Thanks! good catch!